src/pages/Address/AddressComp.tsx

```diff
@@ -287,20 +287,24 @@ export const AddressOverviewCard: FC<{ address: Address }> = ({ address }) => {
                 key={AssetInfo.INSCRIPTION}
               >
                 <div className={styles.assetCardList}>
-                  {inscriptions.map(inscription => {
-                    switch (inscription.udtType) {
-                      case 'omiga_inscription':
-                        return (
-                          <AddressOmigaInscriptionComp
-                            account={inscription}
-                            key={`${inscription.symbol + inscription.udtType + inscription.udtAmount}`}
-                          />
-                        )
-
-                      default:
-                        return null
-                    }
-                  })}
+                  <div className={styles.inscriptions}>
+                    <ul>
+                      {inscriptions.map(inscription => {
+                        switch (inscription.udtType) {
+                          case 'omiga_inscription':
+                            return (
+                              <AddressOmigaInscriptionComp
```

Comment:
`Address Component` should be wrapped by `<li></li>` because it's in `<ul></ul>`